### PR TITLE
Add country code in html lang

### PR DESCRIPTION
**Pull request by gustaveen**
## What?
Add country code in html lang

## Why?
Signal to google that `en` pages are market specific

**Ticket(s): [GRW-1790]**

[GRW-1790]: https://hedvig.atlassian.net/browse/GRW-1790?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/l10n/locales.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ type MarketName =
   | 'Danmark'
   | 'Denmark'
 type LanguageLabel = 'Sv' | 'En' | 'No' | 'Da'
-type HtmlLang = 'sv' | 'en' | 'no' | 'da'
+type HtmlLang = 'sv-se' | 'en-se' | 'no-no' | 'en-no' | 'da-dk' | 'en-dk'
```
Comment (robinandeer) [inline src/l10n/locales.ts:12]: I'm not sure if it makes a difference but shouldn't country part be upper case?

https://www.techonthenet.com/js/language_tags.php

**Review by robinandeer — DISMISSED** [review]
Nice, long time coming!

Just curious, what's the difference between HTML lang and HREF lang? Do we need both?

File: src/l10n/locales.ts (hunk 1 above)
Comment (gustaveen) [reply src/l10n/locales.ts]: Not sure it matters either but let's play it safe :D

**Comment by gustaveen** [comment]
> Nice, long time coming!
> 
> Just curious, what's the difference between HTML lang and HREF lang? Do we need both?

I think it might be good to keep them apart since they might differ. :)

**Pull request merged**